src/cow-react/api/gnosisProtocol/api.ts

```diff
@@ -587,6 +591,23 @@ export async function getNativePrice(chainId: ChainId, address: string): Promise
   }
 }
 
+// TODO: won't be necessary once SDK is integrated
+function transformEthFlowOrder(order: OrderMetaData): OrderMetaData {
+  const { ethflowData, creationDate } = order
+
+  if (!ethflowData) {
+    return order
+  }
+
+  const expireIn5s = Math.ceil((new Date(creationDate).getTime() + 5 * 1000) / 1000)
+  const { userValidTo: validTo } = ethflowData
+  console.log('modified validTo', expireIn5s, { validTo, creationDate })
```

Comment:
Tip: print also the orderId ;)

```suggestion
  console.log('modified validTo', order.uid.slice(0, 10), expireIn5s, { validTo, creationDate })
```

This is my local
![Screen Shot 2022-11-24 at 14 38 43](https://user-images.githubusercontent.com/43217/203810202-c5fab3be-8bd2-44e5-a3aa-c9ef8f005bfd.png)

The time is different because those are different orders 🤷